client/src/main/java/io/pravega/client/batch/SegmentRange.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.batch;
+
+import com.google.common.annotations.Beta;
+import io.pravega.client.batch.impl.SegmentRangeImpl;
+
+/**
+ * This is used to represent range bounded portion of a Segment.
```

Comment:
This is odd because there is nothing in this interface related to bounding a segment. I would expect to see something related to offsets.

Reply:
updated the interface to include the following two APIs.
```
long getStartOffset();
long getEndOffset();
```